qi-doc/scribblings/qi.scrbl

```diff
@@ -9,6 +9,13 @@
 
 An embeddable, general-purpose language to allow convenient framing of programming logic in terms of functional @tech{flows}. A flow is a function from inputs to outputs, and Qi provides compact notation for describing complex flows.
 
+@; Modified from Maciej Barc's req package
+@(let ([logo-path
+        "scribblings/assets/img/logo.svg"])
+   (if (file-exists? logo-path)
+       (centered (image logo-path #:scale 0.7))
+       (printf "[WARNING] No ~a file found!~%" logo-path)))
```

Comment:
I wonder if `define-runtime-path` works here?

Reply:
I'm not totally clear on what this does, but regardless, I've used it here!

Reply:
It has a few key differences; for one, it implies that a built distribution of the documentation will include the logo file.